src/server.ts

```diff
@@ -195,13 +197,8 @@ export class Server {
     return next();
   }
 
-  private async submitResults(ctx: Koa.Context) {
-    const bytesSent = this.currentRunBytes;
-    this.currentRunBytes = 0;  // Reset for next run.
-
+  private async callback(ctx: Koa.Context) {
```

Comment:
`callback` is way less clear of a name than `submitResults`. Is there more that we can say about what this does?

Reply:
Reverted name. I was trying to be consistent with the term "callback", since that's what the CLI flag is. But I agree it's ambiguous in the code.